src/main/scala/tile/RocketTile.scala

```diff
@@ -75,68 +76,27 @@ class RocketTile(
   val itimProperty = tileParams.icache.flatMap(_.itimAddr.map(i => Map(
     "sifive,itim" -> frontend.icache.device.asProperty))).getOrElse(Nil)
 
-  val cpuDevice = new SimpleDevice("cpu", Seq("sifive,rocket0", "riscv")) {
+  val cpuDevice: SimpleDevice = new SimpleDevice("cpu", Seq("sifive,rocket0", "riscv")) {
     override def parent = Some(ResourceAnchors.cpus)
     override def describe(resources: ResourceBindings): Description = {
       val Description(name, mapping) = super.describe(resources)
       Description(name, mapping ++ cpuProperties ++ nextLevelCacheProperty ++ tileProperties ++ dtimProperty ++ itimProperty)
     }
 
+    /**
+      * This function is for backwards compatiblity and will be removed in the future
+      *
+      * @param resourceBindingsMap
+      * @return
+      */
```

Comment:
Are you going to remove this as part of this PR? Why is it necessary to keep it?

Instead of having:
`RocketSubsystem. getOMRocketCores` call 
`RocketTile.cpuDevice.getOMComponents` which makes a private `RocketLogicalTree` and then calls
 `RocketLogicalTree.getOMComponents` on it,
couldn't you just have `RocketSubsystem. getOMRocketCores` call
`rocketTiles.flatMap(c => c.rocketLogicalTree.getOMComponents(resourceBindingsMap))` directly?